meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report

```diff
@@ -1,5 +1,34 @@
 #!/usr/bin/env bash
 
+print_information()
```

Comment:
@esesyn01 @kotylamichal we would also like here to have informations if the logs are empty/with errors/clean. We could use functions similar to https://github.com/Dasharo/meta-dts-ce/blob/init-ce-dell-deploy/meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy#L8 to inform user what was dumped

Reply:
@TomaszAIR done https://github.com/Dasharo/meta-dts-ce/pull/9/commits/3c0e875398d082074092ca8f3f47cab46d14d1e3